packages/paste-core/components/separator/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {render} from 'react-dom';
+import renderer from 'react-test-renderer';
+import {render as testRender, cleanup} from '@testing-library/react';
+import {axe} from 'jest-axe';
+import {Theme} from '@twilio-paste/theme';
+import {Separator} from '../src';
+
+describe('Separator', () => {
+  it('should render', () => {
+    const {getByRole} = testRender(<Separator orientation="horizontal" />);
+    expect(getByRole('separator')).not.toBeNull();
+    cleanup();
```

Comment:
Do we need `cleanup()` here? It looks like that happens automatically with Jest. https://testing-library.com/docs/react-testing-library/api#cleanup

Reply:
Excellent find! Removing